packages/moonstone/Dropdown/Dropdown.js

```diff
@@ -249,6 +308,11 @@ const DropdownBase = kind({
 	},
 
 	handlers: {
+		onKeyDown: handle(
+			forward('onKeyDown'),
+			(ev, props) => forKey('left', ev, props) || forKey('right', ev, props),
```

Comment:
I think we'll need a bit more logic to handle this safely. For example, if you hover a paging control and press 5-way left (or right in RTL), the dropdown will close but should focus an item.